doc/api/events.md

```diff
@@ -1361,6 +1372,26 @@ changes:
       The `EventTarget` class is now available through the global object.
 -->
 
+### `eventTarget[events.maxEventTargetListenersSymbol]([n])`
```

Comment:
This can be removed/not-exposed now right?

Reply:
Yeah that's just a left over. Removing.